tests/Makefile

```diff
@@ -19,79 +16,108 @@ all: minikube operator test
 # Start minikube
 .PHONY: minikube
 minikube:
-	minikube start --driver=docker
-	helm repo update
```

Comment:
Why was this command removed?

Reply:
It was removed in favor of the update flag on adding the helm chart, which is generally a better spot for it in case someone is messing with the helm chart version like I was. That way it uses the correct chart on every operator install.